ui/frontend/src/components/RouteDetails.vue

```diff
@@ -24,6 +24,17 @@
             :inputHandler="updateField"
             :validator="v(uniqueName)"
           />
+          <PropertyField
```

Comment:
please see
https://github.com/aquasecurity/postee/blob/main/ui/frontend/src/components/OutputDetails.vue#L159-L163
no need to define special handlers here

Reply:
changed it